ETC.tsv

```diff
@@ -19582,171 +19582,171 @@ ETC_20151224_019583
-ETC_20151224_019717
-ETC_20151224_019718
-ETC_20151224_019719
-ETC_20151224_019720
-ETC_20151224_019721
-ETC_20151224_019722
-ETC_20151224_019723
-ETC_20151224_019724
-ETC_20151224_019725
-ETC_20151224_019726
-ETC_20151224_019727
-ETC_20151224_019728
-ETC_20151224_019729
-ETC_20151224_019730
-ETC_20151224_019731
-ETC_20151224_019732
-ETC_20151224_019733
-ETC_20151224_019734
-ETC_20151224_019735
-ETC_20151224_019736
-ETC_20151224_019737
-ETC_20151224_019738
-ETC_20151224_019739
-ETC_20151224_019740
-ETC_20151224_019741
-ETC_20151224_019742
-ETC_20151224_019743
-ETC_20151224_019744
-ETC_20151224_019745
-ETC_20151224_019746
-ETC_20151224_019747
-ETC_20151224_019748
-ETC_20151224_019749
-ETC_20151224_019750
-ETC_20151224_019751
+ETC_20151224_019587	Mining...
+ETC_20151224_019588	You cannot mine here.
+ETC_20151224_019589	Only guild members within guilds with the mining skill can mine.
+ETC_20151224_019590	Failed to mine ore.
+ETC_20151224_019591	Ore sites are all depleted
+ETC_20151224_019592	Current members online {Cur}/{Max}
+ETC_20151224_019593	Use the skill: {SkillName}?
+ETC_20151224_019594	The guild master has used the skill: {SkillName}. Adhere to the order?
+ETC_20151224_019595	Guild Master
+ETC_20151224_019596	{PC} is already affiliated with another guild.
+ETC_20151224_019597	You must change to the {Name} class to establish a guild.
+ETC_20151224_019598	{Price} silver has been charged to establish the guild.
+ETC_20151224_019599	Guild cannot be dissolve during while at war
+ETC_20151224_019600	Crops can be grown, and companions can be raised.
+ETC_20151224_019601	War against the {GuildName} guild has started.
+ETC_20151224_019602	War against the {GuildName} guild has ended.
+ETC_20151224_019603	Unable to use the current roadmap 
+ETC_20151224_019604	Guild event area
+ETC_20151224_019605	Only usable when near a guild tower.
+ETC_20151224_019606	Remaining ticket : {Ticket}
+ETC_20151224_019607	Ticket
+ETC_20151224_019608	Select the character receiving the Egg of Rebirth
+ETC_20151224_019609	Withered plants. Unable to harvest them. Clear them out.
+ETC_20151224_019610	{Count} units of the following kind of building can be constructed.
+ETC_20151224_019611	Shield Charge
+ETC_20151224_019612	You lack a shield.
+ETC_20151224_019613	A field boss will appear shortly.
+ETC_20151224_019614	Cannot purchase item registered to the same account.
+ETC_20151224_019615	Selling time has expired. Please check if you recieved your item.
+ETC_20151224_019616	Will register in market after {TIME}.
+ETC_20151224_019617	Will move to cabinet after {TIME}.
+ETC_20151224_019618	To register item, a minimum of {Price} silver is needed.
+ETC_20151224_019619	Registering the following item{nl}will reduce its potential by one.{nl} Do you accept?
+ETC_20151224_019620	Humanoid
+ETC_20151224_019621	Use {TP}TP to purchase {nl}Character/Companion Slot?
+ETC_20151224_019622	Use {TP}TP to change your Team Name?
+ETC_20151224_019623	Not enough Character/Companion Slot.{nl}
```

Comment:
You do not have enough Character/Companion slots.{nl}

or

Insufficient Character/Companion slots.{nl}